webcompat/static/css/src/labels.css

```diff
@@ -175,3 +175,27 @@
 .label-box .label-editor-list [class *= "label-status"] {
   border-left-color: var(--label-closed);
```

Comment:
the default should be `var(--label-closed)` but it is `var(--label-fourth)` 🤔

Reply:
I set `--label-closed` now as the default color.